### DISCUSSION TITLE
<Help with Wallpapers>

### DISCUSSION THREAD

**Discussion by artytux** [community/community#40162] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello
I'd like to contribute to a project here on GitHub (if Works out have more to follow)
I'm at best  just a graphics person and would like to help out with some artwork,
all I'd like to do is have some artwork up for grabs for distros maintainers
on a page here on GitHub is that possible,
am I looking in the right place ?
or am I better to go to Neocities or similar anyone got any suggestions, I know very little about hosting a website so any help as to where or who to go to would be great because all this seems quite an epic.
I hope someone feels kind enough to help.
Regards
artytux

**Comment by airtower-luna** [comment]
> all I'd like to do is have some artwork up for grabs for distros maintainers
> on a page here on GitHub is that possible,

You can use [Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) to host your site, if you like.

I'm not sure if "here's some art for grabs" is the most efficient way to contribute, though. If you make artwork without discussing it with project maintainers first, what you have might be good, but not what they need. So you might want to talk to them first. Of course being able to show the kind of work you do might help with that. :wink:

**Comment by jfullstackdev** [comment]
speaking of the themes, check this out

https://github.com/topics/jekyll-theme

you can even contribute to them

**Comment by artytux** [comment]
@airtower-luna 
The 'pages' links at a quick look it  seems that's what I'm wanting to do.

> I'm not sure if "here's some art for grabs" is the most efficient way to contribute, though. If you make artwork without discussing it with project maintainers first, what you have might be good, but not what they need. So you might want to talk to them first. Of course being able to show the kind of work to do might help with that.

I wrote in a quick explain version, I didn't want to over explain and waste folks time and yes that's what I would have been doing, working with what the maintainers want and or need,
Work with them to make graphics that highlight the features they most proud or happy with in their distro not just dump artwork that they would cringe at.

@jdevstatic 
I wasn't too sure about getting just one option, now I have two, 
I'll have check  this jecyll-theme out carefully,
Spoilt  with choices 

Taking that first step and asking what seemed to me a stupid question, so many clever people here,
well I had to ask, stuff the embarrassed feeling.

Thank You both for your kind replies

Kind Regards
artytux

**Comment by artytux** [comment]
Hello there
I can run almost any Linux distro and fix it when things go wrong, I look at those setups for a website pages and I'm lost there is no way I understand what is happening or what I'm to do next,
I'm not a help vampire and having to have some-one direct and help me with every step of the procedure, no way, thanks but no thanks.
With the best of intentions you all try to help, here is example of --
--  example - landing on the page to make a web page  there seems to be no steps to follow that make sense,
just type this in and a web page will be generated easy eh uh what I want to know what it would look like, -a preview maybe.
Then on another page there's a video great a video that will help.
No audio just follow the cursor going so quickly around and around clicking this and clicking that - no option to slow down the video - what - cannot slow down video useless.
please the reason I switched over to watch the video was to find out what these 3 easy steps were to setup a web page.
Sure the person making the video knows the how to do - does the person watching for the first time umm probably a big NO -
 slow down video - no option - very disappointing.

Helping  me who is not used to and unfamiliar with all the jargons and language used here would similar to me telling about somethings I'm very familiar with (( the servicing of steam boilers and the safety valves before a government safety audit ) yes I'm old ))  and in a very quick and almost abstract style explain to you the how to do,  have you not  understood also you have to do the work, as I instructed you quickly.
How would cope with that mm ?

What chance do you have of completing the task, when someone is unfamiliar about (pick any) topic it is best to explain things at their level.
No help  here, have to be familiar and able to understand the language jarsons etc.

I 'm very upset that there such a flippant attitude, this has completely turned me cold very cold, the artwork stays with me, too bad if some-one any-one wants to copies of the first batch, next time ( after the holidays ) I'm making a second batch just for me !

The behavior from people saying easy peasy setup your website , that  is absurd, previous mentioned behavior,
might be knowledgeable certainly not useful at passing on knowledge.

Respectfully Cold
artytux
